src/ogc-api/model.ts

```diff
@@ -54,6 +54,10 @@ export interface OgcApiCollectionInfo {
   license?: string;
   queryables: CollectionParameter[];
   sortables: CollectionParameter[];
+  // will be empty if the endpoint does not support tiles
+  mapTileFormats: MimeType[];
+  vectorTileFormats: MimeType[];
+  supportedTileMatrixSet: string[]; // identifiers
```

Comment:
```suggestion
  supportedTileMatrixSets: string[]; // identifiers
```
I think this property name would be more descriptive